### ISSUE TITLE
ACC-264.4 (Level AA) _ Modify Modules _ Button _ Information not announced 

### ISSUE THREAD

**Issue by shubha1090** [kyma-project/busola#3152]
1.1 Internal Message Details
Internal Message:	
Test Plan:	ACC SAP Kyma runtime 2024
Message Priority:	Medium
Affected UI	Button
Affected Check Point:	ACC-264.4
Short text:	ACC-264.4 (Level AA) _ Modify Modules _ Button _ Information not announced 

1.2 Application Details
Application Name	Kyma-dashboard
Technology:	SAP UI5 
Component:	
URL:	https://dashboard.stage.kyma.cloud.sap/cluster/garden-kyma-stage--c-7074971-external/kymamodules
Username/Password:	NA
1.3 Test Environment Details
Web browser:	Google Chrome Version 127.0.6533.89
Operating System:	Microsoft Windows 11 x64
Screen Reader:	JAWS 2024 Version 2024.2406.121 ILM

1.4 JIRA/SNOW Details
Issue:	Bug
Labels:	Accessibility

2 Screen Flow
•	Launch the application using given URL. 
•	Select the ‘Link to Kyma dashboard’ from the Kyma Environment section on your SAP BTP subaccount page. Now Select ‘Modify Modules’ button which opens ‘Modules’ page.
•	Open JAWS 2024 screen reader and turn on VPC mode.
•	Activate ‘Add’ button from the main region and select any module and activate ‘Add’ button again to add a module.
•	Now activate ‘Delete’ button from the main region to delete the added module.

3. Issue Type: Manual
Observed Behavior:
On activating ‘Add’/’Delete’ module a user can add/delete a particular module to the list and the success message ‘Module was added/deleted’ is displayed to the user, however this success information i.e. ‘Module was added/deleted’ is not conveyed to the  screen reader user. 
Screenshot:
 
 
 
Case 2:
 
Case 3:
 

Expected Behavior:
On activating ‘Add’/’Delete’ module a user can add/delete a particular module to the list and the success message ‘Module was added/deleted’ is displayed to the user, and  this success information i.e. ‘Module was added/deleted’ must be announced by the screen reader along with other necessary information. 

________________________________________________________________________
Kindly refer to attached error and issue processing document for more details.
Regards,
Shubha

**Comment by shubha1090** [comment]
[ACC-264.4 (Level AA) _ Modify Modules _ Button _ Information not announced.docx](https://github.com/user-attachments/files/16660186/ACC-264.4.Level.AA._.Modify.Modules._.Button._.Information.not.announced.docx)

[Issue processing-09.2021 (002).docx](https://github.com/user-attachments/files/16660187/Issue.processing-09.2021.002.docx)
